### ISSUE TITLE
Problem importing QGFied : from hn2016_falwa.oopinterface import QGField

### ISSUE THREAD

**Issue by AminFazlKazemi** [csyhuang/hn2016_falwa#43]
Hi. Dr. Huang

This is Amin Fazl Kazemi. Sorry for sending lots of issues
I sent you an email for issue : "Problem using qgfield_object.compute_lwa_and_barotropic_fluxes()" 

Hope hear the answer. 

I also faced problem installing the package in windows. I nstalled Ubuntu 21 linux over vmware workstation hyper visor and problem is solved. Thanks in advance

**Comment by csyhuang** [comment]
I don't have a Windows machine so I don't think I can solve this Windows on my side. I will open a ticket regarding the Windows issue, but will wait for volunteer to solve it.

For your issues with the QGField, I searched my email and realized I actually missed that when I travel. Sorry about that. Has your issue got solved (in another ticket you mentioned you computed reference states, so I assumed it is solved...?)?
If you have no issues running through `qgfield_object.compute_lwa_and_barotropic_fluxes()`, I will close this ticket in 2 days.
